cypress/integration/util_spec.js

```diff
@@ -0,0 +1,38 @@
+describe('SMK Util Tests', () => {
+    it('Has a non-empty SMK.UTIL object', () => {
+        cy.window().its('SMK.UTIL').should('not.be.null');
+    })
+
+    it('Should get meters per millimeter', () => {
+        let util;
+        cy.window()
```

Comment:
I tried a few ways to pull out this code since it's repeated in the test below, but I don't think I understand Promises well enough to get it working. I also found that cy.window() cannot be called outside an "it" block.